tests/app/googleanalytics/test_pixels.py

```diff
@@ -0,0 +1,72 @@
+import app.googleanalytics.pixels as gapixels
+
+import pytest
+import urllib
+
+from flask import current_app
+from app import GovdeliveryClient
+
+
+@pytest.fixture(scope='function')
```

Comment:
Replace with a mock provider (there are already example of that in the existing tests)

Reply:
used the mock provider in the root conftest